src/EnergyPlus/ExteriorEnergyUse.hh

```diff
@@ -53,77 +53,85 @@
 
 // EnergyPlus Headers
 #include <EnergyPlus/DataGlobals.hh>
-#include <EnergyPlus/Data/EnergyPlusData.hh>
 #include <EnergyPlus/EnergyPlus.hh>
 
 namespace EnergyPlus {
 
-namespace ExteriorEnergyUse {
-
-    // Using/Aliasing
-
-    // Data
-    // MODULE PARAMETER DEFINITIONS:
-    extern int const ElecUse;         // Electricity
-    extern int const GasUse;          // Gas (Natural)
-    extern int const WaterUse;        // Water
-    extern int const CoalUse;         // Coal
-    extern int const FuelOil1Use;     // FuelOil#1
-    extern int const FuelOil2Use;     // FuelOil#2
-    extern int const PropaneUse;      // Propane
-    extern int const GasolineUse;     // Gasoline
-    extern int const DieselUse;       // Diesel
-    extern int const SteamUse;        // Steam
-    extern int const DistrictCoolUse; // Purchased Cooling
-    extern int const DistrictHeatUse; // Purchased Heating
-    extern int const OtherFuel1Use;   // OtherFuel1
-    extern int const OtherFuel2Use;   // OtherFuel2
-
-    extern int const ScheduleOnly;       // exterior lights only on schedule
-    extern int const AstroClockOverride; // exterior lights controlled to turn off during day.
-
-    // DERIVED TYPE DEFINITIONS:
+struct EnergyPlusData;
+struct ExteriorEnergyUseData;
 
-    // MODULE VARIABLE DECLARATIONS:
+namespace ExteriorEnergyUse {
 
-    // SUBROUTINE SPECIFICATIONS FOR MODULE <module_name>
+    enum class ExteriorFuelUsage {
```

Comment:
Here is where I converted all those `int const` into an enumeration.  As I said, this isn't part of the actual scope here, so don't sweat every one of these, but in either case, leave them declared in the original header.